src/Tribe/Service_Providers/Context.php

```diff
@@ -270,6 +270,38 @@ static function ( $post_type ) {
 					Tribe__Context::REQUEST_VAR => [ TEC::TAXONOMY ],
 				],
 			],
+			'view_url' => [
+				'read'  => [
+					Tribe__Context::FUNC => [
```

Comment:
:warning: Array double arrow not aligned correctly; expected 10 space(s) between "FUNC" and double arrow, but found 1.

<sub>hash: ec501d66947b6033d6fa640522c4f65d</sub>